scripts/upsert-experience.js

```
@@ -0,0 +1,125 @@
+ * Upserts an Experience in Strapi by slug (create if missing, update if exists).
+ * Requires STRAPI_API_TOKEN and optionally STRAPI_URL (default http://localhost:1337).
+ * Loads .env from repo root and apps/cms.
+ *
+ * Usage:
+ *   node scripts/upsert-experience.js <slug> [locale] [payload.json]
+ *   echo '{"slug":"home","isHomepage":true}' | node scripts/upsert-experience.js home en
+ *
+ * Payload: { slug, isHomepage?, sections? }. Sections: dynamic zone array of
+ *   { __component: "sections.media-collection"|"sections.promo-banner"|"sections.info-blocks"|"sections.cta", ...attrs }
+ */
+const fs = require("fs")
```

ESLint

[error] A `require()` style import is forbidden.

(`@typescript-eslint/no-require-imports`)

---

[error] 'require' is not defined.

(no-undef)